Forge/src/main/java/me/ferdz/placeableitems/client/ClientListener.java

```diff
@@ -1,27 +1,79 @@
 package me.ferdz.placeableitems.client;
 
+import me.ferdz.placeableitems.PlaceableItems;
 import me.ferdz.placeableitems.client.model.GlassBottleFluidModel;
 import me.ferdz.placeableitems.client.renderer.tileentity.FluidHolderRenderer;
+import me.ferdz.placeableitems.event.ItemPlaceHandler;
 import me.ferdz.placeableitems.init.PlaceableItemsBlockRegistry;
+import me.ferdz.placeableitems.network.CNotifySpecialItemPlaceKeyPacket;
+import me.ferdz.placeableitems.network.PlaceableItemsPacketHandler;
 import me.ferdz.placeableitems.tileentity.FluidHolderTileEntity;
 
+import net.minecraft.client.settings.KeyBinding;
 import net.minecraft.util.LazyLoadBase;
 
+import net.minecraftforge.client.event.InputEvent;
 import net.minecraftforge.eventbus.api.SubscribeEvent;
 import net.minecraftforge.fml.client.registry.ClientRegistry;
 import net.minecraftforge.fml.event.lifecycle.FMLClientSetupEvent;
 
+import org.lwjgl.glfw.GLFW;
+
 public final class ClientListener {
 
     private static final LazyLoadBase<ClientListener> INSTANCE = new LazyLoadBase<>(ClientListener::new);
 
-    private ClientListener() { }
+    private static final KeyBinding KEY_BINDING_PLACE_SPECIAL_ITEM = new KeyBinding("key." + PlaceableItems.MODID + ".place_special_item", GLFW.GLFW_KEY_LEFT_SHIFT, "key.categories." + PlaceableItems.MODID);
```

Comment:
Why `SPECIAL`? I feel like `KEY_BINDING_PLACE_ITEM` is better. Could you go and rename them in the ids too (`place_special_item`)?